tldr.py

```diff
@@ -196,6 +205,16 @@ def get_page(command, remote=None, platforms=None, languages=None):
         platforms = get_platform_list()
     if languages is None:
         languages = get_language_list()
+    # only use cache
+    for platform in platforms:
```

Comment:
Let's wrap a `if USE_CACHE:` around this so as to avoid wasted cycles on entering the function and exception handling if we don't need it.